.github/workflows/core_build.yml

```diff
@@ -5,8 +5,8 @@ on:
       - 'master'
   pull_request:
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+concurrency: 
```

Comment:
```suggestion
concurrency:
```

Reply:
@Kitzunu You were too quick :). I got this updated now.